### PR TITLE
ref: prefer qualified imports

### PR DESCRIPTION
**Pull request by edvardm**
Prefer qualified imports. 

Also remove unused import I introduced in my previous PR

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -150,3 +150,4 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 .idea/
+Pipfile/
```
Comment (andreoliwa) [inline .gitignore:153]: Curiosity: why this?

**Review by andreoliwa — APPROVED** [review]

**Review by andreoliwa — CHANGES_REQUESTED** [review]
I merged #2 and I created a conflict. 😬

File: .gitignore (hunk 1 above)
Comment (edvardm) [reply .gitignore:153]: I was certain I wrote reason somewhere, maybe missed. It seems like running tests creates this file? Though cleaner way would be to do teardown and remove it there, but this was 'quick fix'. I don't think I put it there, so I thought some tests might write that file?

**Comment by edvardm** [comment]
Conflict should be now resolved, but for some reason Pipfile appeared there again

File: .gitignore (hunk 1 above)
Comment (andreoliwa) [reply .gitignore:153]: > It seems like running tests creates this file?

That's weird.
`Pipfile` is from pipenv, and it's not a directory. 🤔 

It should be something on your compuer, it doesn't happen here.

I just checked out your branch:
```
$ poetry run pytest
================================================================================================ test session starts ================================================================================================
platform darwin -- Python 3.9.7, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /Users/aa, configfile: pytest.ini
collected 10 items

tests/test_conjuring.py .........x    
```

How are you running tests?

Comment (andreoliwa) [reply .gitignore:153]: > Though cleaner way would be to do teardown and remove it there

Remove it where?
There are no Pipfile tests as far as I remember.

Did you forget to push commits?

Comment (andreoliwa) [reply .gitignore:153]: @edvardm ☝🏻

Comment (edvardm) [reply .gitignore:153]: Ok, this is really odd. But totally trust it's something in my system then -> removing .gitignore entry

**Review by andreoliwa — APPROVED** [review]

**Pull request merged**